src/anderson_darling.jl

```diff
@@ -30,6 +30,15 @@ immutable OneSampleADTest <: ADTest
     A²::Float64 # Anderson-Darling test statistic
 end
 
+"""
+    OneSampleADTest{T<:Real}(x::AbstractVector{T}, d::UnivariateDistribution)
```

Comment:
You could do both if `T` isn't used in the body of functions, but to keep this PR focused better only touch docstrings.